File: app/src/main/java/com/demon/activitychange/ui/fragment/MyScriptFragment.java
package com.demon.activitychange.ui.fragment;

import android.graphics.Color;

import com.demon.activitychange.ui.presenter.MyScriptPresenter;
import com.lixh.base.BaseFragment;
import com.lixh.view.UToolBar;


public class MyScriptFragment extends BaseFragment<MyScriptPresenter> {

    @Override
    public void initTitle(UToolBar toolBar) {
        toolBar.setTitle("小爱");
        toolBar.setTitleTextColor(Color.WHITE);
    }

    @Override
    public int getLayoutId() {
        return 0;
    }
}

File: app/src/main/java/com/demon/activitychange/ui/fragment/MyScriptFragment.kt
package com.demon.activitychange.ui.fragment

import android.os.Bundle
import com.demon.activitychange.R
import com.lixh.base.BaseFragment

class MyScriptFragment : BaseFragment(ui = {
    titleBar {
        title = "小爱"
    }
    body(R.layout.fragment_mine)
}) {
    override fun init(savedInstanceState: Bundle?) {

    }

}
